From 32868d3db1294a7facfd23f209b031c29e376788 Mon Sep 17 00:00:00 2001
From: Eugen Hristev <eugen.hristev@collabora.com>
Date: Mon, 29 May 2023 17:24:26 +0300
Subject: [PATCH 17/28] spl: Kconfig: fix trivial typo `Suppport`

s/Suppport/Support

Signed-off-by: Eugen Hristev <eugen.hristev@collabora.com>
---
 common/spl/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/spl/Kconfig b/common/spl/Kconfig
index 2c042ad306..c66b38ee67 100644
--- a/common/spl/Kconfig
+++ b/common/spl/Kconfig
@@ -1373,7 +1373,7 @@ config SYS_USB_FAT_BOOT_PARTITION
 	  Partition on the USB storage device to load U-Boot from
 
 config SPL_USB_GADGET
-	bool "Suppport USB Gadget drivers"
+	bool "Support USB Gadget drivers"
 	help
 	  Enable USB Gadget API which allows to enable USB device functions
 	  in SPL.
-- 
2.39.2

